### ISSUE TITLE
Ability to disable Traefik

### ISSUE THREAD

**Issue by RothAndrew** [defenseunicorns/zarf#68]
As a user of Zarf Appliance Mode, I'd like the ability to disable Traefik so that it doesn't conflict with Istio if I'm deploying Big Bang using Zarf.

**Comment by RothAndrew** [comment]
@jeff-mccoy I'm thinking this can be done with no code changes to Zarf, just a new item in `features:` in the init zarf.yaml and moving traefik from `local.manifests` to the new feature. Thoughts?

**Comment by jeff-mccoy** [comment]
Yeah I've thought about this for a while, except that we do integrate some logic for the ingress cert as part of the default flow and that would be kind of funky if traefik wasn't included.  Hmmm...

**Comment by RothAndrew** [comment]
I'm wondering how this will work now that the required docker registry needs Traefik to work.

What if we added a new flag to change the HTTP(S) ports that Traefik runs on? Would that be super complicated or not bad?
